avatar.css

```diff
@@ -0,0 +1,35 @@
+.avatar{
+  border-radius: 50%;
+  border: 2px solid #fff;
+}
+
+.avatar-text{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color:var(--primary-color);
+}
```

Comment:
in line no.-3 and line no.-10 you are using color with different types . it will break consistency. you should use either hex code or color with varable name